toolchain/check/import.h

```diff
@@ -11,6 +11,12 @@
 
 namespace Carbon::Check {
 
+struct AddImportNamespaceResult {
+  SemIR::NameScopeId name_scope_id;
+  SemIR::InstId inst_id;
+  bool is_duplicate_of_namespace_in_current_package;
+};
```

Comment:
Thoughts on moving this to be adjacent to `AddImportNamespace`? Since it's a free function, I think it'd be helpful to have the return struct and function be adjacent.

Reply:
Done.